operations/configuration_management/CFS_Key_Management.md

```diff
@@ -0,0 +1,33 @@
+## CFS Key Management and Permission Denied Errors
+
+Configuration Framework Service \(CFS\) manages it's own keys separate from keys for communication between CFS and the components or images that it is configuring.  These are separate from the keys used by users and should not need to be managed.
+
+### Permission Denied Errors
+
+If Ansible is unable to connect with it's target and fails with an `Unreachable - Permission denied` error the first place to check is the `cfs-state-reporter` on the target node.
+
+Every booted node should be running a copy of cfs-state-reporter.  This service is responsible for pulling down the public key.  To check the status of this service, ssh to the node that CFS can't communicate with, and run `systemctl status cfs-state-reporter`. 
+
+The cfs-state-reporter should be complete, but report success.  Any other state
+can be an indication of a problem.
+
+```
+ncn-m001:~ # systemctl status cfs-state-reporter
+● cfs-state-reporter.service - cfs-state-reporter reports configuration level of the system
```

Comment:
We are now separating input/output into different code blocks. The format should be changed to something similar to the following:

```
ncn-m001# systemctl status cfs-state-reporter
```

Example output:

```
● cfs-state-reporter.service - cfs-state-reporter reports configuration level of the system
   Loaded: loaded (/usr/lib/systemd/system/cfs-state-reporter.service; enabled; vendor preset: disabled)
   Active: inactive (dead) since Wed 2022-01-19 18:53:45 UTC; 1s ago
  Process: 678311 ExecStart=/usr/bin/python3 ${MODULEFLAG} ${MODULENAME} (code=exited, status=0/SUCCESS)
 Main PID: 678311 (code=exited, status=0/SUCCESS)
 ```

Reply:
fixed